kong/plugins/aws-lambda/aws-serializer.lua

```diff
@@ -103,8 +114,8 @@ return function(ctx, config)
     local requestTime = date(start_time):fmt("%d/%b/%Y:%H:%M:%S %z")
     local requestTimeEpoch = start_time * 1000
 
-    -- Kong does not have the concept of stage, so we just let resource path be the same as path
-    local resourcePath = path
+    local resourceId = route and route.id
+    local resourcePath = resource
```

Comment:
I actually think that the comment that you removing is valid.
A resource is nothing but the HTTP request path and we should not be changing it. The AWS docs page that is linked in this message also has HTTP request path as the resource. Is there a good reason for changing it?

Can we instead mimic the same behavior across traditional and ATC router?
Expressions field is not useful at all to the end-user or AWS.